content/docs/command-reference/plot/show.md

```diff
@@ -0,0 +1,130 @@
+# plot show
+
+Generate a [plot](/doc/command-reference/plot) from a file with
+[continuous metrics](/doc/command-reference/plot#continous-metrics).
+
+## Synopsis
+
+```usage
+usage: dvc plot show [-h] [-q | -v] [-t [TEMPLATE]] [-r RESULT] [--show-json]
+                     [-f FIELDS]
+                     [datafile]
+
+positional arguments:
+  datafile              Data to be visualized.
```

Comment:
```suggestion
  datafile              Continuous metrics file to visualize
```
Cc @pared for the core repo PR

p.s. should this term use singular or plural "metric(s)" ?

Reply:
Moved to iterative/dvc/pull/3577

Reply:
metrics - plural in many docs. let's use plural.

Reply:
This has been applied in the core repo, so we can do it here too 🙂

Reply:
I don't like `continuous`, my doubts are summarized in my first comment.

Reply:
Agree, but that seems to be the paragraph in the core repo right now so I'm going to commit this for now. We can update it again when needed.

Reply:
Ah never mind I see its `Metrics file to visualize.` now. Will update 🙂

Reply:
Btw extra period 😅

Reply:
p.s. please see iterative/dvc/pull/3759 @pared